src/lib/core/DataModelTypes.h

```diff
@@ -43,8 +43,9 @@ typedef uint32_t FieldId;
 typedef uint16_t ListIndex;
 typedef uint32_t TransactionId;
 
-static constexpr FabricIndex kUndefinedFabricIndex = 0;
-static constexpr EndpointId kInvalidEndpointId     = 0xFFFF;
-static constexpr ListIndex kInvalidListIndex       = 0xFFFF; // List index is a uint16 thus 0xFFFF is a invalid list index.
+constexpr FabricIndex kUndefinedFabricIndex = 0;
```

Comment:
Since these are in a header, I believe `static` may be appropriate here (I think compiler may not be required to initialize these at compile time and hide them away, especially in debug builds)

Reply:
Yes, but `constexpr` implies `const` and global `const` implies internal linkage, so `static` is just redundant in this case.